report-ng/app/package.json

```diff
@@ -39,10 +40,13 @@
         "aurelia-templating": "^1.10.4",
         "aurelia-typed-observable-plugin": "^0.4.2",
         "aurelia-ui-virtualization": "^1.0.0-rc2",
+        "echarts": "5.4.2",
         "highlight.js": "^11.8.0",
         "moment": "^2.29.4",
         "moment-duration-format": "^2.3.2",
-        "t-systems-aurelia-components": "^1.1.0",
+        "pixelmatch": "^5.2.1",
+        "protobufjs": "^6.11.3",
```

Comment:
`pixelmatch` and `protobufjs` have been added and not updated?

Reply:
`pixelmatch` and `protobufjs` are now updated.

Reply:
No I mean, where does this dependency comes from in the first place?
`pixelmatch` and `protobufjs` aren't used before, why are they now?

Reply:
You are right. These dependencies were added with a merge some time ago and are indeed no longer required. They have now been removed.